### PR TITLE
Use pre-built CLDR data

### PR DESCRIPTION
**Pull request by feerrenrut**
- replace cldr submodule with nvda-cldr
- update buildsystem for cldr

### Link to issue number:
Related: #14109 

### Summary of the issue:
The CLDR repository is large (~7GB).
Cloning this repository (even as a shallow clone) takes ~7 mins on Appveyor.
NVDA requires very little of the data.
CLDR data included with NVDA is not updated regularly.

### Description of user facing changes
For developers:
The size of submodules has been reduced significantly.
The time required to clone a new copy of NVDA is reduced.
The time for appveyor builds to complete has been reduced.

### Description of development approach
A new repository is used to produce the CLDR data required by NVDA.
See https://github.com/nvaccess/nvda-cldr

The 'main' branch contains the submodule and a script to produce the output.
The 'main-out' branch is updated via a GitHub workflow, it contains only the output from the script.

NVDA now uses the `nvda-cldr` repository as a submodule instead of the `cldr` repository, the `main-out` branch is checked out.
NVDA's buildscript merely copies the files to the `source/locale` directory.

### Testing strategy:
Build on appveyor.
Build locally.
Run NVDA locally and read entries from the `source/locale/cldr.dic`

### Known issues with pull request:
None

### Change log entries:
None

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

**Review by seanbudd — APPROVED** [review]

File: sconstruct [hunk 1]
```diff
@@ -284,6 +284,7 @@ for t2tFile in env.Glob(os.path.join(userDocsDir.path,'*','*.t2t')):
 			styles,
 			installedStyle
 		])
+
```
[the remark this answers is not in the document]
Comment (feerrenrut) [reply sconstruct:287]: It's easier to read with the newline. It also helps to tie the diff for the PR together, the context for this change shows where the CLDR data comes from.

**Comment by cary-rowen** [comment]
I get the following error using ```git submodule update```, is this expected?

fatal: 'nvaccess' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: Fetched in submodule path 'include/nvda-cldr', but it did not contain 130d93caecfe812dbe6bb96bc9d1d3ebc2349552. D
irect fetching of that commit failed.

**Comment by seanbudd** [comment]
@cary-rowen - Does `git submodule sync --recursive` and `git submodule update --init --recursive` fix this?

**Comment by cary-rowen** [comment]
@seanbudd 
I did ```git submodule update --init --recursive``` and it didn't fix the problem, same output.

**Pull request merged**